libres/lib/analysis/update.cpp

```diff
@@ -1,7 +1,8 @@
 #include <vector>
 #include <string>
 #include <assert.h>
-#include <optional>
+#include <fmt/format.h>
```

Comment:
Why these includes?

Reply:
because of https://github.com/equinor/ert/blob/6df137c28285ea9f0d495a207939cc636a8e6cf1/libres/lib/analysis/update.cpp#L493

The compiler decided to just ignore the missing include :shrug: